testing/postgres-client-tests/node/workbenchTests/index.js

```diff
@@ -17,9 +24,25 @@ export default async function runWorkbenchTests(database) {
 async function runTests(database, tests, name) {
   console.log("Running tests for", name);
   await Promise.all(
-    tests.map((test) => {
+    tests.map(async (test) => {
       if (test.skip) return;
 
+      if (test.file) {
+        const filePath = path.resolve(testDataPath, test.file);
+        try {
+          // TODO: Is it possible to test the COPY FROM output?
```

Comment:
Good question... the server does return the number of rows inserted, so would be cool to test that if the pg-copy-streams library exposes that somehow.

Reply:
Tried figuring this out and didn't get very far. Will revisit in another PR